PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkJobLogConsoleView.java

```diff
@@ -45,7 +44,7 @@ public SparkJobLogConsoleView(@NotNull Project project) {
         super(project, true);
 
         // set `usePredefinedMessageFilter = false` to disable predefined filter by console view and avoid filter conflict
```

Comment:
>  // set `us [](start = 6, length = 12)

seems need to remove this line to ConsoleViewWithMessageBars.kt

Reply:
Clean up it later. 😄